### ISSUE TITLE
Add set show/ hide controls possibility

### ISSUE THREAD

**Issue by eiler21** [brianegan/chewie#551]
Please add `set show/ hide controls` possibility. Like the one in the [better_player](https://pub.dev/packages/better_player) .

**Comment by vietstone-ng** [comment]
Yes, I know there is `showControls`, but it is only set in the constructor. How to toggle its value while the player is on screen?
